IdentityCore/src/util/NSDictionary+MSIDExtensions.m

```diff
@@ -94,4 +94,22 @@ - (NSDictionary *)dictionaryByRemovingFields:(NSArray *)fieldsToRemove
     return mutableDict;
 }
 
+- (NSArray<NSURLQueryItem *> *)urlQueryItemsArray;
```

Comment:
just curious, where is this being used?
and nit: there's a ";" in the end

Reply:
we use this constructing startURL.
```
urlComponents.queryItems = [parameters urlQueryItemsArray];
```